docs/education/sessions.md

```diff
@@ -0,0 +1,111 @@
+## Fabric Token SDK - Session 1 - 29.10.2025
```

Comment:
I think you can remove both date and `Presenter`.
It is very nice to have this in form of sessions or lessons.